media-video/ffmpeg/ffmpeg-4.0.ebuild

```diff
@@ -350,8 +351,8 @@ multilib_src_configure() {
 	# Incompatible features: openssl and gnutls
 	# openssl support provides a (strict) superset of gnutls support as of 2017.11.30
 	# So, we warn the user and disable gnutls
-	if use openssl && use gnutls; then
-		ewarn "openssl and gnutls are mutually exclusive in ${PN}, disabling gnutls since openssl provides more features"
+	if use openssl && use gnutls || use openssl && use libressl || use libressl && use gnutls; then
```

Comment:
I think USE="openssl libressl" will trigger the warning and fail at configure

maybe:
```
if use openssl; then
    if use libressl; then
        ewarn  ...
        myconf+=( --disable-libtls )
    fi
    if use gnutls ; then
        ....
    fi
fi
```

Reply:
note that you also need to define a default for USE='-openssl gnutls libressl'

Reply:
Yes, USE="libressl openssl" will trigger a block due to the libressl ebuild.  So I adjusted that.  I also adjusted the warnings for disabling the various providers.